src/main/java/sg/ncl/MainController.java

```diff
@@ -170,6 +171,15 @@ public String jointeam() {
 //    public String resource2() {
 //        return "resource2";
 //    }
+    @RequestMapping("/accessexperiment")
+    public String accessexperiment() {
+        return "accessexperiment";
+    }
+
+    @RequestMapping("/resource2")
+    public String resource2() {
+        return "resource2";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)